### PR TITLE
docs: update aspect compile

### PR DESCRIPTION
**Pull request by gene-zhan**
fix error Command

### CODE REVIEW

**Review by luketheart — APPROVED** [review]

**Review by luketheart — APPROVED** [review]
LGTM.

File: docs/develop/guides/reentrancy-guard/index.md [hunk 1]
```diff
@@ -182,43 +182,78 @@ If the reentrant attack succeeded, you will see both `AddLiquidity` and `RemoveL
...
+            const callTreeQuery = new CallTreeQuery(-1);
+
+            const queryCallTree = sys.hostApi.trace.queryCallTree(callTreeQuery);
+            const ethCallTree = Protobuf.decode<EthCallTree>(queryCallTree, EthCallTree.decode);
+            var size = ethCallTree.calls.size;
+            var arrayKeys = ethCallTree.calls.keys();
+
+            for (let i = 0; i < size; i++) {
+                var key = arrayKeys[i];
+                var oneCall = ethCallTree.calls.get(key);
```
Comment (dumbeng) [inline docs/develop/guides/reentrancy-guard/index.md:226]: oneCall -> parentCall

File: docs/develop/reference/aspect-lib/join-points/verify-aspect/index.md [hunk 2]
```diff
@@ -193,3 +193,222 @@ sys.log('block.header.number' + ' ' + numberData.data.toString(10));
...
+
+To trigger the 'VerifyTx' join point, a customized verification transaction without a signature is necessary. The format
+of this transaction is as follows:
+
+```shell
+magic prefix + checksum(encodedData) + encodedData(validation data + raw calldata)
+```
+
+**Description：**
+> * 0xCAFECAFE serves as a stable value with a magical prefix.
```
Comment (dumbeng) [inline docs/develop/reference/aspect-lib/join-points/verify-aspect/index.md:207]: stable value -> fixed vaue

File: docs/develop/reference/aspect-lib/join-points/verify-aspect/index.md [hunk 3]
```diff
@@ -193,3 +193,222 @@ sys.log('block.header.number' + ' ' + numberData.data.toString(10));
...
+
+```shell
+0xCAFECAFE75bac07d00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000
+0000000000000000000000000c000000000000000000000000000000000000000000000000000000000000000557d7cfdeff97cbabdc4124c779d51
+bb2520cbdc769840c6fee4566e23c7c22ea2a3e071dc794283b70608de4566c9dc0fd5f1854d7032676996fd95284693ecad5b2eeca9864734af5131
+ed015c52cd1b5b3d9f841c000000000000000000000000000000000000000000000000000000000000000000000000000000000000241003e2d2000
+000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000
+```
+
+* 0xCAFECAFE75 is a magical prefix.
```
Comment (dumbeng) [inline docs/develop/reference/aspect-lib/join-points/verify-aspect/index.md:222]: no 75 here

File: docs/develop/reference/aspect-lib/join-points/verify-aspect/index.md [hunk 4]
```diff
@@ -193,3 +193,222 @@ sys.log('block.header.number' + ' ' + numberData.data.toString(10));
...
+* 75bac07d is 4-byte checksum of the encoded data.
```
Comment (dumbeng) [inline docs/develop/reference/aspect-lib/join-points/verify-aspect/index.md:223]: Need to explain how to calculate the checksum

File: docs/develop/reference/aspect-lib/join-points/verify-aspect/index.md [hunk 5]
```diff
@@ -193,3 +193,222 @@ sys.log('block.header.number' + ' ' + numberData.data.toString(10));
...
+* The remaining bytes represent the result of ABI encoding for ValidationData and CallData.
```
Comment (dumbeng) [inline docs/develop/reference/aspect-lib/join-points/verify-aspect/index.md:224]: Need to explain how to encode the data

File: docs/develop/reference/aspect-lib/join-points/verify-aspect/index.md [hunk 6]
```diff
@@ -193,3 +193,222 @@ sys.log('block.header.number' + ' ' + numberData.data.toString(10));
...
+
+### Create customized verification transaction
```
Comment (dumbeng) [inline docs/develop/reference/aspect-lib/join-points/verify-aspect/index.md:226]: This part is too long, the validation data constructed here is for session key, we need to provide a minimum case here.

**Review by dumbeng — CHANGES_REQUESTED** [review]

**Review by dumbeng — APPROVED** [review]
LGTM

**Pull request merged**